eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -68,6 +68,24 @@ jobs:
 
       - script: |
           node common/scripts/install-run-rush.js install
+          node common/scripts/install-run-rush.js lint $(rushFlags)
+        displayName: 'Run lint'
```

Comment:
Does this fail the analyze step if there are lint errors, or does this step always succeed and just generate reports?

Reply:
with `exit 0` it always succeeds. If we remove that, it will fail with exit code 1 if there are lint errors in the reports.

Reply:
We should run these as two separate steps. On Windows I have noticed problems when you run multiple commands in the same script step.